docs/integrations/saas-cloud/microsoft-exchange-tracelogs.md

```diff
@@ -0,0 +1,134 @@
+```sql title="Message Delivery Status"
+_sourceCategory="o365dummy"
+| json "Organization", "MessageId", "Received","SenderAddress", "RecipientAddress", "Subject", "Status", "ToIP", "FromIP", "Size", "MessageTraceId", "StartDate", "EndDate", "Index" as organization, message_id, received, sender_address, recipient_address, subject, status, toIP, fromIP, size, message_traceId, start_date, end_Date, index nodrop
+| where status matches"{{status}}"
+| where sender_address matches"{{sender_address}}"
+| where organization matches "{{organization}}"
+| lookup latitude, longitude, country_code, country_name, region, city, postal_code from geo://location on ip = fromIP
+| where country_name matches "{{country}}"
+| timeslice 1d
+| count by _timeslice, status
+| fillmissing timeslice, values all in status
+| transpose row _timeslice column status
+```
+
+## Collecting logs from Microsoft Exchange Tracelogs App
+
+This section provides instructions for setting up [Cloud-to-Cloud-Integration for Microsoft Exchange Tracelogs App](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-exchange-tracelogs/) to create the source and use the same source category while installing the app.
+
+## Installing the Gmail Trace Logs app
+
+Locate and install the app from the **App Catalog**. If you want to see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
+
+To install the app, follow the steps below:
+1. From the **App Catalog**, search for the app and select it.
+2. Select the version of the service you're using and click **Add to Library**.
+:::note
+Version selection is applicable only to a few apps currently. For details, see the [Install the Apps from the Library](/docs/get-started/apps-integrations#install-apps-from-the-library).
+:::
+3. To install the app, enter the following fields.
+   1. **App Name**. You can retain the existing name, or enter a name of your choice for the app.
+   2. **Data Source**. Select either of these options for the data source:
+      * Choose **Source Category** and then choose a source category from the list.
+      * Select **Enter a Custom Data Filter** and type in a custom source category that starts with an underscore. For Example: `_sourceCategory=MyCategory`.
+    3. **Advanced**. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
+4. Click **Add to Library**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. You can share it with your organization.
+
+The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.
+
+
+## Viewing Dashboards
```

Comment:
Viewing Microsoft Exchange Trace Logs Dashboards

Reply:
Thank you for the suggestion. However, I believe we don’t have to mention the title again. “Viewing Dashboards” looks appropriate.